.circleci/config.yml

```diff
@@ -118,6 +140,26 @@ workflows:
                 - main
 
     jobs:
+      # Ruby 3.1 release
+      - bundle_and_test:
+          name: "ruby3-1_rails6-1"
+          ruby_version: 3.1.2
+          rails_version: 6.1.1
+      - bundle_and_test:
+          name: "ruby3-1_rails6-0"
+          ruby_version: 3.1.2
+          rails_version: 6.0.5
```

Comment:
Ruby 3.1 and Rails 6.0 is not a supproted combination at all on RAils end, so take out this combo.

Reply:
Great.